docs/v0.5.x/installation.md

```diff
@@ -2,8 +2,13 @@
 title: Installation
 ---
 
-To install, run
+ember-cli-deploy is just an Ember CLI addon so it's super easy to install:
 
 ```
 ember install ember-cli-deploy
 ```
+
+But ember-cli-deploy doesn't work just on it's own. You also need to install plugins to actually do the deployment work.
```

Comment:
"But ember-cli-deploy doesn't work just on it's own." -> "But ember-cli-deploy doesn't do much by itself."